### ISSUE TITLE
provider-upload-aws-s3: image upload fills caption and alt text with file name

### ISSUE THREAD

**Issue by sven-ra** [strapi/strapi#13295]
### Required System information

- Node.js version: 16.14.0
- Yarn version: 1.22.17
- Strapi version: 4.1.9
- Database: MySQL
- Operating system: MacOS 12.1

### Describe the bug

I use `@strapi/provider-upload-aws-s3` to upload files to AWS S3. Whenever I upload an image the alternative text and caption fields get filled with file name value.

### Steps to reproduce the behavior

1. Set up `@strapi/provider-upload-aws-s3`
2. Upload an image
3. See the description and alt text field

### Expected behavior

Alternative text and caption fields should be empty

### Screenshots

<img width="832" alt="image" src="https://user-images.githubusercontent.com/16561130/168035330-5e5a457b-81e8-49ed-8a3f-ca3e9e469a9a.png">

**Comment by LucaNerlich** [comment]
Is this really a but? Tbh id rather like these "default values". Better than empty string.

**Comment by gu-stav** [comment]
@LucaNerlich we discussed the attached PR today and think it is valuable to not pre-fill the alternative text with the filename. Why? Accessibility wise all 3 cases are valid:

- omitting the `alt` attribute: if the filename needs to be read out to screenreader users
- leaving `alt` empty: if an image is decorative, the alt text should be empty, because screenreaders would then not read the name of the file
- filling `alt`: for most images an alt text describing the image should be provided

We think that prefilling the alt attribute with the filename promotes a bad accessibility practice. Why do you prefer having the filename as a fallback?

**Comment by Tikondra** [comment]
please remove the auto-completion of the caption

**Comment by cooperwfloyd** [comment]
I'd also appreciate the removal of the default behavior of filling the alt text and the caption with the file name. Many developers that I've worked with on accessibility projects have found through research and from working with accessibility agencies that it is typically worse for accessibility to fill the alt with the file name than to leave it blank.

Would also be helpful to have this data blank by default on the frontend when it comes to conditionals for hiding / showing content based on if the string exists or not. If you want a caption to show on your site, you probably don't want it to be the file name 👍

**Comment by cooperwfloyd** [comment]
Thanks for the quick response here @derrickmehaffy, I haven't contributed to Strapi before but I'd like to give it a shot on this one if I can.

It appears that [these two lines](https://github.com/strapi/strapi/blob/main/packages/core/upload/admin/src/hooks/useUpload.js#L21-L22) are the ones that are setting the `caption` and the `alternativeText` to the file name by default, would it be best to remove the `|| name` on each so that they are `null` by default?

**Comment by gu-stav** [comment]
The issue has been fixed by https://github.com/strapi/strapi/pull/14790 From now on neither caption nor alternative text will be pre-filled with the filename.

Thanks everyone for your contributions and the discussion! 🚀
